src/internal/operations/backup.go

```diff
@@ -306,6 +308,7 @@ func (op *BackupOperation) do(
 		mdColls,
 		lastBackupVersion,
 		op.Options,
+		op.canMakeDeltaQueries,
```

Comment:
similar to the other comment about ownership, passing a GC-produced flag back into GC from the operations layer doesn't make much sense and upsets dependency principles.  Similarly, it muddies clarity between the query result and the forced no-delta option.